sdk/core/Azure.Core.Amqp/api/Azure.Core.Amqp.netstandard2.0.cs

```diff
@@ -29,9 +29,13 @@ public partial class AmqpAnnotatedMessage
     }
     public partial class AmqpMessageBody
     {
+        public AmqpMessageBody(System.Collections.Generic.IEnumerable<System.Collections.Generic.IList<object>> sequence) { }
         public AmqpMessageBody(System.Collections.Generic.IEnumerable<System.ReadOnlyMemory<byte>> data) { }
+        public AmqpMessageBody(object value) { }
```

Comment:
What types are allowed?

Reply:
http://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-types-v1.0-os.html#section-primitive-type-definitions

Reply:
LEt's add some tests to make sure all of these are supported and what .NET types they correspond to.